src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
```

Comment:
you should use the same font-family for this font-faces, but different font-weight